sdk/storage/azure-storage-file/src/test/java/com/azure/storage/file/spock/FileTestHelper.groovy

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.file.spock
+
+import com.azure.core.http.rest.Response
+import com.azure.core.implementation.util.ImplUtils
+import com.azure.core.util.configuration.ConfigurationManager
+import com.azure.storage.file.models.CorsRule
+import com.azure.storage.file.models.FileServiceProperties
+import com.azure.storage.file.models.Metrics
+import com.azure.storage.file.models.RetentionPolicy
+import com.azure.storage.file.models.ShareItem
+import com.azure.storage.file.models.StorageErrorCode
+import com.azure.storage.file.models.StorageErrorException
+
+import java.time.Duration
+
+import static org.junit.Assert.assertEquals
```

Comment:
Should this code be merged into APISpec as that is sort of the helper class for everything?